File: 0.1 Programming basics/Homework 2 Data-Types-And-Variables/10.EmployeeData/EmployeeData.cs
using System;

namespace _10.EmployeeData
{
    class EmployeeData
    {
        static void Main()
        {
            string firstName = "Pesho";
            string lastName = "Georgiev";
            byte age = 27;
            char gender = 'f';
            long personalID = 8306112507;
            int uniqueEmployeeNumber = 27563571;

            Console.WriteLine("First name: {0}\nLast name: {1}\nAge: {2}\nGender: {3}\nPersonal ID: {4}\nUnique Employee Number: {5}",
                firstName,lastName,age,gender,personalID,uniqueEmployeeNumber);
        }
    }
}

File: 4.1 C Programming/Homework 2 C Data Types/EmployeeData.c
/* 
 * Problem 8. Employee Data
A marketing company wants to keep record of its employees. 
 * Each record would have the following characteristics:
 First name
 Last name
 Age (0...100)
 Gender (m or f)
 Personal ID number (e.g. 8306112507)
 Unique employee number (27560000…27569999)
 */

#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

/*
 */ 
int main(int argc, char** argv) {
    char *firstName = "Artur";
    char *lastName = "Chobanski";
    char age = 44;
    char gender = 'm';
    unsigned long long personalIDNumber = 8306112507l;
    int uniqueEmployeeNumber = 27569999;
    
    printf("First Name: %s\n", firstName);
    printf("Last Name: %s\n", lastName);
    printf("Age: %d\n", age);
    printf("Gender: %c\n", gender);
    printf("Personal ID Number: %llu\n", personalIDNumber);
    printf("Unique Employee Number: %d\n", uniqueEmployeeNumber);
    
    return (EXIT_SUCCESS);
}

